config/rhoai/prometheus_rule.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: prometheus-rules
+  namespace: openshift-monitoring
+spec:
+  groups:
+    - name: kueue-critical-alerts
+      rules:
+      - alert: KueuePodDown
+        expr: kube_pod_status_ready{pod=~"kueue-.*", condition="true"} == 0
+        for: 5m
+        labels:
+          severity: critical
+        annotations:
+          summary: "Kueue pod is down ({{ $labels.pod }})"
+          description: "The Kueue pod {{ $labels.pod }} in namespace {{ $labels.namespace }} is not ready."
+    - name: kueue-info-alerts
+      rules:
+      - alert: HostCpuIsUnderutilized
+        expr: (100 - (rate(node_cpu_seconds_total{mode="idle"}[30m]) * 100) < 20) * on(instance) group_left (nodename) node_uname_info{nodename=~".+"}
+        for: 1d
+        labels:
+          severity: info
+        annotations:
+          summary: Host CPU is underutilized (instance {{ $labels.instance }})
+          description: "CPU load is below 20% for 1 day. Consider reducing the number of CPUs."
+      - alert:  HostHighCpuLoad
+        expr: (100 - (rate(node_cpu_seconds_total{mode="idle"}[30m]) * 100) > 80) * on(instance) group_left(nodename) node_uname_info{nodename=~".+"}
+        for: 1d
+        labels:
+          severity: info
+        annotations:
+          summary: Host high CPU load (instance {{ $labels.instance }})
+          description: "CPU load is above 80% for 1 day. Consider increasing the number of CPUs"
+      - alert: ContainerLowMemoryUsage 
+        expr: (sum(container_memory_working_set_bytes{name!=""}) BY (instance, name) / sum(container_spec_memory_limit_bytes > 0) BY (instance, name) * 100) < 20
+        for: 1d
+        labels:
+          severity: info
+        annotations:
+          summary: Container Low Memory Usage (instance {{ $labels.instance }})
+          description: Container memory usage is below 20% for 1 day. Consider reducing available memory.
```

Comment:
These are useful, but I'm not sure these are appropriate as Kueue alerts

Reply:
Do you think it is sufficient just to have the resource usage compared to quota ones for kueue then? I don't think I can do similar to this using kueue queries, and I think it would be useful to know that the resources you are using are either being underused and may not be needed or are high and you need more. I'm thinking of this from a metrics UI point of view, where alert functionality will be added.

Reply:
If these or similar metrics aren't surfaced elsewhere then I would add them, but I feel that these should be platform level alerts. It just feels outside our responsibility to be alerting about over/under utilization from Kueue.

Reply:
I have removed them, you're right they don't belong in kueue. Will follow up to include them somewhere else for the metrics UI alerts functionality. I've made an extra commit with these changes for now but will squash before it's ready to merge.